docs-ref-conceptual/install-azure-cli-windows.md

```diff
@@ -144,6 +145,18 @@ In order to get the MSI, your proxy needs to allow HTTPS connections to the foll
 * `https://aka.ms/`
 * `https://azcliprod.blob.core.windows.net/`
 
+### Migrate to 64-bit Azure CLI
+
+Starting from 2.51.0, Azure CLI also provides 64-bit MSI which is recommended for better performance.
+
+If you are using 32-bit version, you can migrate to 64-bit version by following these steps:
+1. Check your current CLI version and installed extensions by running `az --version`.
+2. Extensions will need to be reinstalled. It is recommended to perform a backup of the current extension folder `%userprofile%\.azure\cliextensions` by renaming it in case you need to revert to the 32-bit version. This folder will be created again after migration.
+3. Download and install latest 64-bit version following the [Install or update](#install-or-update) guide. Please note that 32-bit MSI will be uninstalled automatically.
+4. Install extensions by running `az extension add --name <extension> --version <version>`.
```

Comment:
```suggestion
1. Install extensions by running `az extension add --name <extension> --version <version>`.  The Azure CLI also prompts you to install an extension upon first use.
```

Reply:
Could you please provide more details about the prompt?